styles/style.css

```diff
@@ -270,9 +328,11 @@ label {
 }
 
 /* Buttons */
-.menu {
+.menu-btn {
   background: none;
   border: none;
+  position: relative;
+  z-index: 1;
 }
 
```

Comment:
- [ ] It would be better if we adding instructions about how to clone and run the app locally in our README file :+1: This will make non-technical people also make use of the project

Reply:
**Hi @addod19 👋

Thank you very much for the feedback 👏, all points you mentioned are very clear and I have already started working on them 🐱‍🏍.

However, There is only one thing that I didn't really understand which is your request to update the live demo.

 I insert the live demo link below and it seems working to me, so please would you explain that more to me ?!

 **[live demo](https://adel-gu.github.io/portfolio-mobile-first/)**